src/fhirtypes/StructureDefinition.ts

```diff
@@ -88,6 +91,7 @@ export class StructureDefinition {
    * A StructureDefinition instance of StructureDefinition itself.  Needed for supporting escape syntax.
    */
   private _sdStructureDefinition: StructureDefinition;
+  private originalMapping: StructureDefinitionMapping[] = [];
```

Comment:
We're a little inconsistent with our use of `_` prefixed private properties in SUSHI -- but I wonder if we should use it here (e.g., `_originalMapping`) since the only other private property uses it -- and since the private `_original` in ElementDefinition.ts uses it as well.

Reply:
So this is a little bit of a silly reason for this, but I didn't use `_originalMapping` because StructureDefinitionExporter actually strips all properties that start with `_` in order to clear out things that do not apply to the new structure definition that is being built (lines [436-443](https://github.com/FHIR/sushi/blob/master/src/export/StructureDefinitionExporter.ts#L436-L443)). However, we do still want to keep track of the original mapping even on the new structure definition. I could add a separate check there to not clear out `_originalMapping` specifically, but I thought that seemed slightly less clean than just using `originalMapping` instead.

Reply:
Oh, I see. That's a bit of an unintentional side-effect of FHIR also using `_` for primitive shadow properties. Got it. In that case, let's just leave it as is. Thanks!